contrib/stack/stripmapStack/prepareUAVSAR_coregStack.py

```diff
@@ -14,15 +15,15 @@ def createParser():
     Create command line parser.
     '''
 
-    parser = argparse.ArgumentParser(description='Unzip Alos zip files.')
+    parser = argparse.ArgumentParser(description='Prepare UAVSAR SLC Stack files.')
     parser.add_argument('-i', '--input', dest='input', type=str, required=True,
-            help='directory which has all dates as directories. Inside each date, zip files are expected.')
+            help='directory which has all dates.')
     parser.add_argument('-d', '--dop_file', dest='dopFile', type=str, required=True,
-            help='Doppler file for the stack.')
+            help='Doppler file for the stack. Needs to be in directory where command is run.')
     parser.add_argument('-o', '--output', dest='output', type=str, required=True,
             help='output directory which will be used for unpacking.')
-    parser.add_argument('-t', '--text_cmd', dest='text_cmd', type=str, default='source ~/.bash_profile;'
```

Comment:
It might be useful to change the default to ''. Having this command in front is useful when different jobs are being submitted via a queuing system on a cluster. Could be used to setup the right env / entrypoint.

Reply:
Thanks. I think I overwrote that "--text_cmd" parameter in my editing. Would be good to add it back. Do I need to do that in my pull request?

Reply:
Sorry, it seems that I switched to a different branch when I committed the last change.